lib/arborist/reify.js

```diff
@@ -99,6 +98,7 @@ module.exports = cls => class Reifier extends Ideal(cls) {
       scriptShell,
       savePrefix = '^',
       binLinks = true,
+      save = true,
```

Comment:
As much as it's good to be explicit about what we're pulling off the options object at each stage, since this variable isn't actually used in this scope, I think it'd be better to remove this line.

Reply:
Yeah, I concur with that. I'll pull it out.